bower.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "CarLog",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "dependencies": {
-    "angular": "1.x.x",
-    "bootstrap": "3.x.x",
+    "angular": "1.3.14",
```

Comment:
Really? You know that it will select the highest version available when you give it an x in the minor or patch portions of the version.